src/component/Board.js

```diff
@@ -1,10 +1,20 @@
 import React from "react";
+import Race from "./race/Race";
 import Text from "./text/Text";
 import TypingBox from "./typingbox/TypingBox";
 import Timer from "./Timer";
 import WordsPerMinute from "./WordsPerMinute";
 
 function Board(props) {
+  function renderRace() {
+    return (
+      <Race
+        wordsLength={props.words.length}
+        currentWordIndex={props.currentWordIndex}
+      />
+    );
+  }
+
```

Comment:
⚒ I'm not a super big fan of these functions that don't anything but render a component, but this is more-so my stylistic preference; I like to avoid extraneous functions unless they significantly improve readability, which I don't think is the case here.

This could be a separate PR if we choose to make the change, as we have a few other instances of this.

Reply:
The change would be to do all of the rendering in the return statement of the `Board` component? I'm open to everything, as I don't know many React mannerisms, if it makes more sense not to have these extraneous functions then I'm happy to change it. We could even move everything into the `Game` component and delete `Board`.

Reply:
Will get to this in a different PR